recipes/glow/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "glow" %}
+{% set pypi_name = "glow.py" %}
+{% set version = "0.6.0.post1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://test.pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.tar.gz
+  sha256: 32736e84c2c3df8f62bf5afaaebe6d96f09228bb27f2d953d2b736de7a9a082d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.7
+    - pyspark >=2.4.0
+    - nptyping >=1.3.0
+    - numpy >=1.17.4
+    - opt_einsum >=3.2.0
+    - pandas >=0.25.3
+    - statsmodels >=0.10.0
+    - typeguard >=2.9.1
+
+test:
+  requires:
+    - pyspark
```

Comment:
It would be useful to add hail as a test dependency to make sure that we can use `glow.hail`.

Reply:
I'm not sure if we can install packages from sources other than `conda-forge`. I'll give it a try.

Reply:
You can't, though perhaps you'd like to add `hail` to conda-forge as well so it can be included as a runtime dependency? It should be straight-forward using `grayskull`: https://github.com/conda-forge/staged-recipes#grayskull---recipe-generator-for-python-packages-on-pypi

(even if it's optional, conda-forge packages tend to be fairly "complete" if the dependencies aren't too large)